docs/event/command.md

```diff
@@ -19,17 +19,13 @@ How can we use an [Event Streaming Platform](../event-stream/event-streaming-pla
 ## Solution
 ![Command](../img/command-event1.svg)
 
-Separate out the function call into a service that writes an event to
-an [Event Stream](../event-stream/event-stream.md), detailing the
-necessary action and its arguments. Then write a separate
-service that watches for that event before invoking the
-procedure.
+One service can invoke a function call in another service through the use of an [Event Stream](../event-stream/event-stream.md). The producer service (Application A) creates a command detailing the action for the consumer service (Application B) to take. The command also includes any necessary supporting information for the execution of the command.
 
-In terms of application logic, a Command is typically dispatched in a fire-and-forget manner (the actual record representing the command is delivered and stored with strong guarantees, such as exactly-once semantics).  The writer assumes that the command will be handled correctly, and the responsibility for monitoring and error-handling falls elsewhere in the system.  This is very similar to the Actor model: actors have an inbox, and we write messages to that inbox and trust that they will be handled in due course.
+In terms of application logic, a command is typically dispatched in a fire-and-forget manner. The actual record representing the command is produced to the [Event Stream](../event-stream/event-stream.md) and stored with strong durability guarantees. The writer assumes that the command will be handled correctly by the appropriate subscriber service, and the responsibility for monitoring and error-handling falls elsewhere in the system. This is very similar to the Actor model: actors have an inbox, messages are written to that inbox, and they are trusted to process the messages in due course.
```

Comment:
I think this would feel better to me if we just tweaked that first sentence. Something like:

> Ideally, commands are issued without expecting a response, beyond a simple acknowledgment of receipt.

Reply:
Yep, good call out. I'll keep it as per your second sentence. If we want to revisit this to overhaul it, I think then would be a good opportunity to get into more detail.